windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -2,14 +2,29 @@
 # SPDX-License-Identifier: BSD-2-Clause
 ---
 # Description:
-# Install inbox version of WSL
-
 # Install WSL and distribution
-- name: "Install WSL and distribution"
+#
+# Due to known issue for store version of WSL(https://github.com/microsoft/WSL/issues/9231),
+# "Launching WSL from session zero does not currently work (for example from an ssh connection)",
+# and the support of inbox version is removed from Windows build 25267, the test case will install inbox version for
+# build lower than 25267 until the known issue is resolved.
+- name: "Install inbox version of WSL and distribution"
   include_tasks: ../utils/win_execute_cmd.yml
   vars:
     win_powershell_cmd: "wsl --install --inbox -d {{ wsl_distribution_name }} | Out-File -FilePath {{ wsl_file_path_win }}\\wsl_install.txt"
     win_execute_cmd_ignore_error: true
+  when:
```

Comment:
`win_powershell_cmd` can be set to different values depends on build number instead of include tasks 2 times.

Reply:
Fixed